### PR TITLE
Splash screen issue fixed

### LINKED ISSUE #3: Gif on splash screen does not render properly
**Describe the bug**
The gif pauses for a few seconds before the next screen is shown, which is not good user experience.

**To Reproduce**
1. Open InstiGo app.
2. Observer when the splash screen is there.

**Expected behavior**
The animation must not lag and the transition from splash screen to other screens should be smooth.

**Screenshots**
gif_lags

**Smartphone (please complete the following information):**
 - Device: Redmi 4
 - OS: Andorid Nougat 7.1
 - Version: Build 1.0.0

### PR DESCRIPTION
**Pull request by rob729**
Fixes #3 

<a href="https://imgflip.com/gif/34oedk"></a>

### CODE REVIEW

**Comment by rob729** [comment]
@sonusourav Please review this PR.

**Comment by rob729** [comment]
I have used a different approach in this PR.

File: app/src/main/res/anim/fade_out.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (sonusourav) [inline app/src/main/res/anim/fade_out.xml]: The transition looks good, but you could put integer 0 instead of this file (R.anim.fade_out).

**Review by sonusourav — CHANGES_REQUESTED** [review]

**Comment by rob729** [comment]
@sonusourav I have done the requested changes.

File: app/src/main/res/anim/fade_out.xml (hunk 1 above)
Comment (sonusourav) [reply app/src/main/res/anim/fade_out.xml]: you can delete this right?

**Review by sonusourav — CHANGES_REQUESTED** [review]

**Comment by rob729** [comment]
Deleted fade_out.xml

**Pull request merged**